app/code/Magento/CustomerImportExport/Model/ResourceModel/Import/CustomerComposite/Data.php

```diff
@@ -28,16 +28,18 @@ class Data extends \Magento\ImportExport\Model\ResourceModel\Import\Data
      *
      * @param \Magento\Framework\Model\ResourceModel\Db\Context $context
      * @param \Magento\Framework\Json\Helper\Data $coreHelper
+     * @param \Magento\ImportExport\Model\ResourceModel\Import\Data\IteratorFactory $iteratorFactory,
      * @param string $connectionName
      * @param array $arguments
      */
     public function __construct(
         \Magento\Framework\Model\ResourceModel\Db\Context $context,
         \Magento\Framework\Json\Helper\Data $jsonHelper,
+        \Magento\ImportExport\Model\ResourceModel\Import\Data\IteratorFactory $iteratorFactory,
```

Comment:
See http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter for a correct way to add new dependency.